src/main/scala/org/broadinstitute/dsde/workbench/avram/db/CollectionComponent.scala

```diff
@@ -0,0 +1,65 @@
+package org.broadinstitute.dsde.workbench.avram.db
+
+import java.sql.Timestamp
+import java.time.Instant
+
+import org.broadinstitute.dsde.workbench.avram.api.Collection
+import AvramPostgresProfile.api._
+
+import scala.concurrent.ExecutionContext
+
+case class CollectionRecord(id: Long,
+                            name: String,
+                            createdBy: String,
+                            samResource: String,
+                            createdTimestamp: Timestamp,
+                            updatedBy: String,
+                            updatedTimestamp: Timestamp)
+
+trait CollectionComponent extends AvramComponent {
+
+  class CollectionTable(tag: Tag) extends Table[CollectionRecord](tag, "collection") {
+    def id =                     column[Long]            ("id",                  O.PrimaryKey, O.AutoInc)
+    def name =                   column[String]          ("name",                O.Unique, O.Length(1000))
+    def samResource =            column[String]          ("sam_resource",        O.Length(1000))
+    def createdBy =              column[String]          ("created_by",          O.Length(1000))
+    def createdTimestamp =       column[Timestamp]       ("created_timestamp",   O.SqlType("TIMESTAMP(6)"))
+    def updatedBy =              column[String]          ("updated_by",          O.Length(1000))
+    def updatedTimestamp =       column[Timestamp]       ("updated_timestamp",   O.SqlType("TIMESTAMP(6)"))
+
+    def * = (id, name, samResource, createdBy, createdTimestamp, updatedBy, updatedTimestamp) <> (CollectionRecord.tupled, CollectionRecord.unapply)
```

Comment:
Here's where I'm thinking that we could convert `java.sql.Timestamp`s to `java.time.Instant`s. One thing I'm not sure of is how (in slick) to write the conversions in both directions.

Reply:
I would like to keep Record and Response classes separate so we can keep it like this

Reply:
This is about converting the row tuple from slick to the `*Record` instance, not `*Record` vs `*Response`. I want to insulate business logic from strangeness with `java.sql.Timestamp`, and from `java.sql` in general.

I can be okay with not addressing this in this PR as long as we consider this (and possibly other points) for a pre-prod TODO list.

Reply:
Oh, it looks like you did make this change. Never mind.